internal/apps/dop/endpoints/endpoints.go

```diff
@@ -279,6 +279,7 @@ func (e *Endpoints) Routes() []httpserver.Endpoint {
 		{Path: "/api/testcases/actions/batch-copy", Method: http.MethodPost, Handler: e.BatchCopyTestCases},
 		{Path: "/api/testcases/actions/batch-clean-from-recycle-bin", Method: http.MethodDelete, Handler: e.BatchCleanTestCasesFromRecycleBin},
 		{Path: "/api/testcases/actions/export", Method: http.MethodGet, Handler: e.ExportTestCases},
+		{Path: "/api/testcases/actions/export", Method: http.MethodPost, Handler: e.ExportAIGeneratedTestCases},
```

Comment:
Bad design.
Use `Get` for export. Maybe you should explicitly declare it's AI related on the url path.

Reply:
Get with request body?